### PR TITLE
update package name 'ak-phrase.js' -> 'ak-phrase'

### LINKED ISSUE #54: update package name in readme
The readme labels the wrong package name. The package name is `ak-phrase` not `ak-phrase.js`. You can check by running:
`npm info ak-phrase.js` vs `npm info ak-phrase`.

```diff
- $ npm install ak-phrasae.js
+ $ npm install ak-phrase
```

and 

```diff
- const sentenceGenerator = require('ak-phrase.js');
+ const sentenceGenerator = require('ak-phrase');
```

This might also be a good chance to update the repo name to reflect the npm package name. Updating the docs count as a patch update:  1.0.3 -> 1.0.4

### PR DESCRIPTION
**Pull request by cdrani**
Fixes #54

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,22 +14,24 @@ It follows the word's order in a sentence, which is the same as the word's colum
...
 
-Please consider donating, if you like my work
+Please consider donating, if you like my work.
 
 ## Install
 
-```
-$ npm install ak-phrasae.js
-```
+> $ npm install ak-phrase
```
Comment (arshadkazmi42) [inline README.md]: I think, using code blocks for showing the commands would be better.
Any thoughts?

**Review by arshadkazmi42 — CHANGES_REQUESTED** [review]
Looks good. Thank you for taking this up
I have left a comment for discussion, we can discuss more on that.

File: README.md (hunk 1 above)
Comment (cdrani) [reply README.md]: Sure. Will update.

**Review by arshadkazmi42 — APPROVED** [review]
🎉

**Comment by arshadkazmi42** [comment]
@cdrani Thank you for all the refactoring. 👍

**Pull request merged**